test/suites/scale/deprovisioning_test.go

```diff
@@ -310,27 +317,31 @@ var _ = Describe("Deprovisioning", Label(debug.NoWatch), Label(debug.NoEvents),
 						defer GinkgoRecover()
 						defer wg.Done()
 
-						env.MeasureDurationFor(func() {
-							// Provide a default selector based on the original provisioner name if one isn't specified
-							selector = assertions.deletedNodeCountSelector
-							if selector == nil {
-								selector = labels.SelectorFromSet(map[string]string{v1alpha5.ProvisionerNameLabelKey: provisionerMap[d].Name})
-							}
-							env.EventuallyExpectDeletedNodeCountWithSelector("==", assertions.deletedCount, selector)
-
-							// Provide a default selector based on the original provisioner name if one isn't specified
-							selector = assertions.nodeCountSelector
-							if selector == nil {
-								selector = labels.SelectorFromSet(map[string]string{v1alpha5.ProvisionerNameLabelKey: provisionerMap[d].Name})
-							}
-							env.EventuallyExpectNodeCountWithSelector("==", assertions.nodeCount, selector)
-							env.EventuallyExpectHealthyPodCount(labels.SelectorFromSet(deploymentMap[d].Spec.Selector.MatchLabels), int(lo.FromPtr(deploymentMap[d].Spec.Replicas)))
-						}, aws.DeprovisioningEventType, multipleDeprovisionersTestGroup, defaultTestName,
-							lo.Assign(map[string]string{aws.TestSubEventTypeDimension: d}, aws.GenerateTestDimensions(assertions.createdCount, assertions.deletedCount, replicasPerNode)))
+						// Provide a default selector based on the original provisioner name if one isn't specified
+						selector = assertions.deletedNodeCountSelector
+						if selector == nil {
+							selector = labels.SelectorFromSet(map[string]string{v1alpha5.ProvisionerNameLabelKey: provisionerMap[d].Name})
+						}
+						env.EventuallyExpectDeletedNodeCountWithSelector("==", assertions.deletedCount, selector)
+
+						// Provide a default selector based on the original provisioner name if one isn't specified
+						selector = assertions.nodeCountSelector
+						if selector == nil {
+							selector = labels.SelectorFromSet(map[string]string{v1alpha5.ProvisionerNameLabelKey: provisionerMap[d].Name})
+						}
+						env.EventuallyExpectNodeCountWithSelector("==", assertions.nodeCount, selector)
+						env.EventuallyExpectHealthyPodCount(labels.SelectorFromSet(deploymentMap[d].Spec.Selector.MatchLabels), int(lo.FromPtr(deploymentMap[d].Spec.Replicas)))
+
 					}(k, v)
 				}
 				wg.Wait()
-			}, aws.DeprovisioningEventType, multipleDeprovisionersTestGroup, defaultTestName, aws.GenerateTestDimensions(totalCreatedCount, totalDeletedCount, replicasPerNode))
+			}, map[string]string{
+				aws.TestCategoryDimension:           multipleDeprovisionersTestGroup,
+				aws.TestNameDimension:               defaultTestName,
+				aws.ProvisionedNodeCountDimension:   strconv.Itoa(totalCreatedCount),
+				aws.DeprovisionedNodeCountDimension: strconv.Itoa(totalDeletedCount),
```

Comment:
They are probably still broken but I think I want to keep this out of the scope of this PR right now since this PR isn't trying to fix these issues.

Reply:
Since we know it's broken, can we just change it to the expected values though? Just looking at the code, the expected counts should be 
`totalCreatedCount: 20` and `totalDeletedCount: 38`